ipyleaflet/leaflet.py

```diff
@@ -814,7 +822,15 @@ class Heatmap(RasterLayer):
     max = Float(1.0).tag(sync=True, o=True)
     radius = Float(25.0).tag(sync=True, o=True)
     blur = Float(15.0).tag(sync=True, o=True)
-    gradient = Dict({0.4: 'blue', 0.6: 'cyan', 0.7: 'lime', 0.8: 'yellow', 1.0: 'red'}).tag(sync=True, o=True)
+    colors = ['blue', 'cyan', 'lime', 'yellow', 'red']
+    values = [0.4, 0.6, 0.7, 0.8, 1.0]
+    dict = {}
+    for i in range(len(values)):
+        dict[values[i]] = colors[i]
+    vmin = values[0]
+    vmax = values[len(values) - 1]
+    gradient = Dict(dict).tag(sync=True, o=True)
+    colormap = LinearColormap(colors, vmin=vmin, vmax=vmax)
```

Comment:
`vmin`, `vmax`, `gradient` and `colormap` should be computed when `values` or `colors` change.

Reply:
This has been tried in commit [93fa3c4](https://github.com/jupyter-widgets/ipyleaflet/pull/1036/commits/93fa3c40e40500cb386fa9fd645d01f2415bd2b5).